Functions/xASL_bids_CreateDatasetDescriptionTemplate.m

```diff
@@ -0,0 +1,105 @@
+function [json] = xASL_bids_CreateDatasetDescriptionTemplate(varargin)
+%xASL_bids_CreateDatasetDescriptionTemplate This script creates a JSON structure which can be saved
+% using spm_jsonwrite to get a dataset_description.json template.
+%
+% FORMAT: [json] = xASL_bids_CreateDatasetDescriptionTemplate(varargin)
+% 
+% INPUT:
+%   Name                - Patient name (CHAR ARRAY, OPTIONAL, DEFAULT = 'Undefined')
+%   BIDSVersion         - BIDS Version (e.g. 1.5.0) (CHAR ARRAY, OPTIONAL, DEFAULT = use xASL_bids_Config to get version)
+%   HEDVersion          - HED Version (CHAR ARRAY, OPTIONAL, DEFAULT = 'Undefined')
+%   DatasetType         - Dataset Type (CHAR ARRAY, OPTIONAL, DEFAULT = 'Undefined')
+%   License             - License (CHAR ARRAY, OPTIONAL, DEFAULT = 'Undefined')
+%   Authors             - Authors (CHAR ARRAY, OPTIONAL, DEFAULT = 'Undefined')
+%   Acknowledgements    - Acknowlegements (CHAR ARRAY, OPTIONAL, DEFAULT = 'Undefined')
+%
+% OUTPUT:
+%   json                - JSON structure which can be saved using spm_jsonwrite (dataset_description.json template)
+%                         
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION:  This script creates a JSON structure which can be saved
+%               using spm_jsonwrite to get a dataset_description.json template.
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% EXAMPLE:      [json] = xASL_bids_CreateDatasetDescriptionTemplate('Test Patient','1.5.0','1.2.3','Test','Test License','Author Names','Thanks');
```

Comment:
My proposition would be to have a single struct on the entry with some correct and some incorrect fields.

Reply:
Check out bcfe3e6.